psd-web/spec/forms/edit_investigation_collaborator_form_spec.rb

```diff
@@ -0,0 +1,140 @@
+require "rails_helper"
+
+RSpec.describe EditInvestigationCollaboratorForm, :with_elasticsearch, :with_stubbed_mailer do
+  let(:user_team) { create(:team) }
+  let(:user) { create(:user, :activated, has_viewed_introduction: true, teams: [user_team]) }
+  let(:investigation) { create(:investigation, assignable: user) }
+  let(:team) { create(:team) }
+  let(:collaborator) do
+    create(:collaborator, investigation: investigation, team: team, added_by_user: user)
+  end
+
+  let(:permission_level) { EditInvestigationCollaboratorForm::PERMISSION_LEVEL_DELETE }
+  let(:message) { "" }
+  let(:include_message) { "false" }
+  let(:params_team) { team }
+
+  let(:params) do
+    {
+      permission_level: permission_level,
+      message: message,
+      include_message: include_message,
+      investigation: investigation,
+      team: params_team,
+      user: user
+    }
+  end
+
+  let(:form) {
+    described_class.new(params)
+  }
+
+  before do
+    collaborator
+  end
+
+  describe "#save" do
+    context "when deleting" do
+      context "when successful" do
+        it "removes collaborator record" do
+          expect { form.save }.to change(Collaborator, :count).from(1).to(0)
+        end
+
+        it "returns true" do
+          expect(form.save).to be true
+        end
+
+        # rubocop:disable RSpec/ExampleLength
+        it "sends email" do
+          form.save
+
+          email = delivered_emails.last
+          aggregate_failures do
+            expect(email.recipient).to eq(team.team_recipient_email)
+            expect(email.personalization_value(:case_id)).to eq(investigation.id)
+            expect(email.personalization_value(:updater_name)).to eq(user.name)
+          end
+        end
+        # rubocop:enable RSpec/ExampleLength
+
+        context "when team has no email" do
+          let(:team) { create(:team, team_recipient_email: nil) }
+          let!(:team_user) { create(:user, :activated, has_viewed_introduction: true, teams: [team]) }
+
+          it "sends email to users" do
+            form.save
+
+            email = delivered_emails.last
+            expect(email.recipient).to eq(team_user.email)
+          end
+        end
+
+        # rubocop:disable RSpec/ExampleLength
+        it "creates activity entry" do
+          form.save
+
+          last_added_activity = investigation.activities.reload.order(:created_at).first
```

Comment:
```suggestion
          last_added_activity = investigation.activities.reload.order(created_at: :desc).find_by!(type: "AuditActivity::Investigation::TeamDeleted")
```
and then remove line 78